### PR TITLE
all: fix a few typos spotted by gospel

### PR DESCRIPTION
**Pull request by mvdan**
(see commit message)

### CODE REVIEW

**Comment by mvdan** [comment]
cc @kortschak

File: interp/handler.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ type ExecHandlerFunc func(ctx context.Context, args []string) error
 // DefaultExecHandler returns the ExecHandlerFunc used by default.
 // It finds binaries in PATH and executes them.
 // When context is cancelled, an interrupt signal is sent to running processes.
-// KillTimeout is a duration to wait before sending the kill signal.
```
Comment (kortschak) [inline interp/handler.go:82]: We should have allowed this. It wasn't found by gospel was it?

Comment (mvdan) [reply interp/handler.go:82]: It was found by gospel, and it did puzzle me for a minute, but it's right. The docs initially referenced a field named `KillTimeout` (capitalized), but we switched to a _parameter_ named `killTimeout` (not capitalized), and we did not update the docs.

Comment (kortschak) [reply interp/handler.go:82]: I'm glad it found it, but I don't understand why (I'll look into it); any initial mismatch of "KillTimeout" would have kicked in the camel splitter to check "Kill" and "Timeout", both of which are standard dictionary words. On tip, it passes that label without any additional information.

Comment (mvdan) [reply interp/handler.go:82]: Ah, this was a branch I had worked on locally a few days ago, so perhaps it picked up the typo before you merged the camel case changes.

Comment (kortschak) [reply interp/handler.go:82]: I'd like to be able to get this back while retaining the camel splitting. There are a couple of ways we can do that; 1. keep a `map[string]string` of identifiers lowercased=>verbatim and if there is an initial spelling mismatch check that before camel splitting or 2. on split retry if a success happens check whether the each success part is case-insensitively matching but case-different and gives a case-insenstivly matching suggestion. 1. is simpler.

**Review by riacataquian — APPROVED** [review]
LGTM 🍪

File: interp/handler.go (hunk 1 above)
Comment (mvdan) [reply interp/handler.go:82]: To close the loop, that was https://github.com/kortschak/gospel/pull/42.

**Pull request merged**